### ISSUE TITLE
Android dev's ListView sample throws exception

### ISSUE THREAD

**Issue by anolivetree** [flutter/website#2322]
https://flutter.io/docs/get-started/flutter-for/android-devs

The first sample code of 'How do I update ListView’s dynamically?' throws exception.

```
I/flutter (23342): The following assertion was thrown building SampleAppPage(dirty, state: _SampleAppPageState#02255):
I/flutter (23342): type 'List<dynamic>' is not a subtype of type 'List<Widget>'
```

Replacing List widgets = []; with List&lt;Widget&gt; widgets = []; fixed the problem on my PC.

**Comment by chalin** [comment]
Could be fixed in the context of #2233.

**Comment by mjohnsullivan** [comment]
Fix added in https://github.com/flutter/website/pull/2324
